cfgov/v1/tests/test_forms.py

```diff
@@ -10,17 +10,6 @@
 
 class TestFilterableListForm(TestCase):
 
-    @mock.patch('v1.forms.FilterableListForm.__init__')
-    @mock.patch('taggit.models.Tag.objects')
-    def test_set_authors_filters_tags_on_pageids(self, mock_tag_objects, mock_init):
```

Comment:
@willbarton Should I be replacing this test with something? The languages are a hardcoded list as opposed to the generated list of the authors filter, so I wasn't sure if they needed a dedicated test.

Reply:
I don't think so? I think removing this is fine.